com.sleepfiles.OSCAR.yaml

```diff
@@ -0,0 +1,69 @@
+######################################################
+# Flatpak manifest example for Open Build Service
+# https://docs.flatpak.org/en/latest/manifests.html
+# Input should be YAML, even though the file can have
+# a .json suffix (JSON is a subset of YAML).
+# Don't use '//' comments!
+######################################################
+
+# Special OBS field because flatpak does not have a version field
+# Default will be '0' if the field is missing.
+#!BuildVersion: 1.3.1
+---
+app-id: com.sleepfiles.OSCAR
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: OSCAR
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
```

Comment:
that's a lot of permissions.

Reply:
Sorry, I got the defaults from another template. I changed it to home,xdg-run. Is that okay? This program needs to read files from SD cards.